DoLah/Models/Calendar.cpp

```diff
@@ -1,35 +1,41 @@
 #include "Calendar.h"
 
 namespace DoLah {
+
 	Calendar::Calendar() {
 	}
 	
 	Calendar::~Calendar() {
-		//TODO
 	}
 
-	void Calendar::addTask(AbstractTask* task) {
-		//TODO
+	std::vector<AbstractTask*> Calendar::getTaskList() {
+		return taskList;
 	}
 
-	void deleteTask(int id) {
-		//TODO
+	void Calendar::addTask(AbstractTask* task) {
```

Comment:
Can we have all argument passed by const reference instead?
Use pointer as argument only when no other choice.